### PR TITLE
GETEX, GETDEL and SET PXAT/EXAT Documentation

### PR DESCRIPTION
**Pull request by nmvk**
Documentation changes corresponding to https://github.com/redis/redis/pull/8327.

### CODE REVIEW

File: commands/getex.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Get the value of `key`. Unlike `GET`, this is a write command and would be rejected by replicas. Return values are similar as `GET.
```
Comment (oranagra) [inline commands/getex.md:1]: i'm not sure the reference to replicas is right here.
in theory replicas can be set to accept writes.
```suggestion
Get the value of `key`. Unlike `GET`, this is a write command. Return values are similar as `GET.
```

File: commands/getex.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+@bulk-string-reply: the value of `key`, or `nil` when `key` does not exist.
+
+@examples
+
+```cli
+SET mykey "Hello"
+GETEX mykey
+
+GETEX mykey "will expire in a minute" EX 60
```
Comment (oranagra) [inline commands/getex.md:23]:
```suggestion
GETEX mykey "will expire in a minute" EX 60
TTL mykey
```

File: commands/getdel.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+@return
+
+@bulk-string-reply: the value of `key`, or `nil` when `key` does not exist.
+
+@examples
+
+```cli
+SET mykey "Hello"
+GETDEL mykey
```
Comment (oranagra) [inline commands/getdel.md:11]:
```suggestion
GETDEL mykey
GET mykey
```

**Review by oranagra — COMMENTED** [review]
added minor improvements to the examples, and i think i'd like to delete the reference to read-only replica.
@itamarhaber WDYT?

**Review by oranagra — APPROVED** [review]

File: commands.json [hunk 4]
```diff
@@ -1871,6 +1871,42 @@
     "since": "2.2.0",
     "group": "string"
   },
+  "GETDEL": {
+    "summary":"Get the value of a key and delete the key.",
```
Comment (itamarhaber) [inline commands.json:1875]:
```suggestion
    "summary":"Get the value of a key and delete the key",
```

File: commands.json [hunk 5]
```diff
@@ -1871,6 +1871,42 @@
...
+      {
+        "name": "key",
+        "type": "key"
+      }
+    ],
+    "since": "6.2.0",
+    "group": "string"
+  },
+  "GETEX": {
+    "summary": "Get the value of a key and optionally set expiration of a key or remove expiration of a key",
```
Comment (itamarhaber) [inline commands.json:1887]:
```suggestion
    "summary": "Get the value of a key and optionally set its expiration",
```

File: commands/getdel.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+Get the value of `key` and delete the key. Return values are similar as `GET`.
```
Comment (itamarhaber) [inline commands/getdel.md:1]:
```suggestion
Get the value of `key` and delete the key.
This command is identical to `GET`, except for the fact that it also deletes the key.
```

File: commands/getex.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+Get the value of `key`. Unlike `GET`, this is a write command. Return values are similar as `GET.
```
Comment (itamarhaber) [inline commands/getex.md:1]:
```suggestion
Get the value of `key` and optionally set its expiration.
`GETEX` is similar to `GET`, but is a write command with additional options.

**Review by itamarhaber — COMMENTED** [review]
LGTM - minor changes

**Pull request merged**